src/infix-eng.ts

```diff
@@ -0,0 +1,16 @@
+
+import { AbstractInfixNotation } from "./infix-abstract";
+// import Decimal from "break_infinity.js";
+
+export class InfixEngineeringNotation extends AbstractInfixNotation {
+  public get name(): string {
+    return "Infix engineering";
+  }
+
+  protected formatMantissa(digit: number):string {
```

Comment:
Codacy found an issue: [expected onespace after colon in call-signature](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)